### DISCUSSION TITLE
ADE 2.0.1

### DISCUSSION THREAD

**Discussion by HermesDDD** [apprenticeharper/DeDRM_tools#1856] [Q&A]
Hey there , I sucessfully removed the DRM for my Kindle and am now trying to do the same for the one I have on ADE 2.0.1. As it would seem tho I am only able to download the version2.0.6 ( Im not sure if this has to do smt. with my location since Adobe wont let me download stuff from the .COM site ) and before I f-up everything I wanted to ask if anyone still has the .exe for ADE.

**Comment by ElleKayEm** [comment]
You're unable to download from here?:  https://www.adobe.com/support/digitaleditions/downloads.html

**Comment by HermesDDD** [comment]
Yeah thats exactly the link I tried ( even send it to some friends since its so small ) but the download wont start. 
Same thing for example on this page : https://www.lehmanns.de/page/adedownload
Also disabled my Antivirus since I thought it might be messing but still nope , didnt change a thing

**Comment by j-howell** [comment]
If you are only planning to use ADE to download books you might want to try the new calibre plugin that can replace it for that function.

[ACSM input plugin](https://www.mobileread.com/forums/showthread.php?t=341975)

**Comment by HermesDDD** [comment]
Will the DRM of the ACSM File also be locked if I download it tho ? Since if I download the Ebook with ADE 4.5 the DRM will be set to 4.5 and I wouldnt  be able to convert it then.
Feels risky downloading the file and then ending up with  a file I cant convert .

**Reply by ElleKayEm** [reply]
I don't imagine that plugin would be able to download an ebook with the hardened Adobe DRM any more than ADE 2.0.1 can.

**Reply by Leseratte10** [reply]
I don't think it can, but I haven't tested it yet. Is there a free book available anywhere in ACSM form from a provider that's known to enforce the new Adobe DRM? Or is there any book shop / provider that has sone cheap book that enforces the new DRM (= can't be loaded with ADE 2) so I could take a look at that new DRM and how it works?

**Reply by ElleKayEm** [reply]
I've haven't heard of a provider that is using it exclusively.  And as far as I've seen, places like Kobo and Google don't seem to be using it at all.  I imagine this is a big part of why it has gone so long without being DeDRM supported.  Lack of availability and need to deal with it.

**Reply by Leseratte10** [reply]
And do you know a provider where I could get a free (or cheap) eBook with the new hard DRM if I do use ADE4.5 (even though it also still supports ADE2)?

**Reply by ElleKayEm** [reply]
No, I haven't seen it.  I've only seen a few issues where that may have been the problem, but never found out the source of the book.

**Comment by HermesDDD** [comment]
I can provide you with a book protected by a new DRM for ADE 4.5 if thats allowed on here

**Reply by ElleKayEm** [reply]
A book that cannot be opened with ADE 2.0.1, but can be read with ADE 4.5 authorized with the same Adobe ID?

**Reply by HermesDDD** [reply]
Yes it cant be opened by ADE 2.0.1 but in 4.5 . Im not sure about the ID because i returend the Ebook but I can still acess the Epub even tho I dont have any rights. Cant tell for sure if anybody else could open it

Also since you mentionted Kobo , if Id buy stuff on there, would DRM Removal method with ADE2.0.1 work ?

**Reply by ElleKayEm** [reply]
Not sure if that would work then.  If you don't mind sharing, what is the source of that book?

Yes, I use ADE 2.0.1 to download my Kobo ebooks and then DeDRM them all the time.

**Comment by HermesDDD** [comment]
Its actually from Kobo, but as I mentioned i refunded the book and its gone on the KoboReader App thingy , but the Epub file I downloaded with ADE thorugh acsm still works. ( I Just checked and I guess I still got the rights on ADE , because I can only open it if Im loged into my Adobe ID )

**Reply by ElleKayEm** [reply]
Huh, I hadn't heard of anyone getting the hardened DRM from Kobo even with ADE 4.5.

**Reply by Leseratte10** [reply]
Yeah, that'd be cool if you could get me such a book. I'm not sure if I'd be able to make the DeDRM compatible with it, but I would like to take a look. I'd need the EPUB file, your account encryption key, and, if possible, the ACSM (even if already linked to your account). 
Not sure what the best way to get that file to me would be, but given that some other issues also have DRMed books attached to them, posting that here for everyone to see might be feasible, then others could take a look at the new DRM, too, if they wanted. If you don't want to do that let me know and I'll post a throwaway email address here. 
So you did download this eBook with ADE 4.5, and ADE 2 can't open it?

(Github really should have some kind of private messaging ...)

**Comment by HermesDDD** [comment]
So I downloaded it firstly with the KoboApp , then got my hands on the Acsm-file on their official page and downloaded it with ADE4.5 and then wanted to remove the DRM with with Calibre 4.2.3  ( DeDRM 6.8.1 and the normal KFX Input ) but it didnt work so I thought its because   I downloaded it with the new ADE and it got the hardened DRM. 
Yeah, as said I returned the Book but still have the Epub , but I cant upload the ZIP-File here for you guys because it keeps saying " is not included in the list" tho it clearly states ZIP is included in the list.

Edit: I googled but Im to dumb to find where to look for the Ecryption Key
